src/switch_version.c

```diff
@@ -35,6 +35,11 @@
  */
 #include <switch.h>
 #include <switch_version.h>
+#undef SWITCH_VERSION_REVISION
```

Comment:
This change should not be commited

Reply:
Actually, it's OK, given that we have` '#define SWITCH_VERSION_REVISION "telv48.0+git~1633101647~20211001T152047~75c4494e69"` in switch_version.c <- This should not be there, I'll add instead commit to remove it.